addons/point_of_sale/tests/test_frontend.py

```diff
@@ -782,10 +782,6 @@ def test_rounding_half_up_cash_and_bank(self):
 
     def test_pos_closing_cash_details(self):
```

Comment:
This test seems to be useless now. Let's replace it with a more relevant test.
Test steps:
1. Open session at 0.00.
2. Sell 50.00 with cash payment.
3. Close session with 49.00
4. Check that a statement.line with ref "Cash difference observed during the counting (Loss) - closing" is created.
5. Check that the difference between expected and counted closing amount is -1.00.

Reply:
For this test, I don't think you need to write a tour. There are examples out there that don't create tours.